package.json

```diff
@@ -23,8 +23,8 @@
     "node": ">=12"
   },
   "dependencies": {
-    "nan": "^2.15.0",
-    "node-gyp-build": "^3.9.0"
+    "nan": "^2.20.0",
+    "node-gyp-build": "^4.8.2"
```

Comment:
@rochdev I couldn't find a way to make it break, it would build and link for me but since the `prebuild` script in `scripts/prebuild.js` doesn't exist in this branch there's some secret code not available for me for testing. I reverted `node-gyp-build` and only bumped `nan`.